src/layout/layout.js

```diff
@@ -74,6 +84,7 @@
     CONTAINER: 'mdl-layout__container',
     HEADER: 'mdl-layout__header',
     DRAWER: 'mdl-layout__drawer',
+    DRAWER_LINKS: 'mdl-layout__drawer a',
```

Comment:
This object is specifically to hold CSS classes used in the system. Not general selectors for any given query statement. Instead write your query selector ( line 189 ) as `document.querySelector('.' + this.CssClasses_.DRAWER + ' a')`. This will re-use the existing class as it should, while not expanding the scope for what this property holds.